Gemfile

```diff
@@ -27,3 +27,4 @@ end
 
 gem 'm'
 gem "localhost", require: false
+gem "ruby_memcheck", "~> 1.0"
```

Comment:
I think this should be guarded similar to `rubocop` above (`PUMA_NO_RUBOCOP`) with `PUMA_NO_MEMCHECK` so we can avoid installing it for regular CI runs

Reply:
I added PUMA_MEMCHECK.